platforms/quorum/charts/quorum-node/templates/node-statefulset.yaml

```diff
@@ -0,0 +1,199 @@
+      app.kubernetes.io/namespace: {{ .Release.Namespace }}
+      app.kubernetes.io/release: {{ .Release.Name }}
+      app.kubernetes.io/managed-by: helm
+  serviceName: {{ include "quorum-node.fullname" . }}
+  volumeClaimTemplates:
+  - metadata:
+      name: data
+      labels:
+        {{- if $.Values.labels }}
+        {{- range $key, $value := $.Values.labels.pvc }}
+        {{- range $k, $v := $value }}
+        {{ $k }}: {{ $v | quote }}
+        {{- end }}
+        {{- end }}
+        {{- end }}
+    spec:
+      accessModes: [ "ReadWriteOnce" ]
+      storageClassName: storage-{{ .Release.Name }}
+      resources:
+        requests:
+          storage: "{{ .Values.storage.size }}"
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}
+        app.kubernetes.io/name: quorum-statefulset
+        app.kubernetes.io/component: goquorum
+        app.kubernetes.io/part-of: {{ include "quorum-node.fullname" . }}
+        app.kubernetes.io/namespace: {{ .Release.Namespace }}
+        app.kubernetes.io/release: {{ .Release.Name }}
+        app.kubernetes.io/managed-by: helm
+        {{- if $.Values.labels }}
+        {{- range $key, $value := $.Values.labels.deployment }}
+        {{- range $k, $v := $value }}
+        {{ $k }}: {{ $v | quote }}
+        {{- end }}
+        {{- end }}
+        {{- end }}
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: {{ .Values.node.goquorum.metrics.pprofport | quote }}
+        prometheus.io/path: "/metrics"
+    spec:
+      serviceAccountName: {{ .Values.global.serviceAccountName }}
+      imagePullSecrets:
+      {{- if .Values.image.pullSecret }}
+        - name: {{ .Values.image.pullSecret }}
+      {{- end }}
+      initContainers:
+{{- if has .Values.global.cluster.provider .Values.volumePermissionsFix }}
+      # fix for minikube and PVC's only writable as root https://github.com/kubernetes/minikube/issues/1990
+      - name: volume-permission-goquorum
+        image: busybox
+        command: ["sh", "-c", "chown -R 1000:1000 /data"]
+        volumeMounts:
+          - name: data
+            mountPath: /data
+        securityContext:
+          runAsUser: 0
+{{- end}}
+      containers:
+        - name: "{{ include "quorum-node.fullname" . }}-quorum"
+          image: {{ .Values.image.goquorum.repository }}:{{ .Values.image.goquorum.tag }}
+          imagePullPolicy: {{ .Values.image.goquorum.imagePullPolicy }}
+          resources:
+            requests:
+              cpu: "{{ .Values.node.goquorum.resources.cpuRequest }}"
+              memory: "{{ .Values.node.goquorum.resources.memRequest }}"
+            limits:
+              cpu: "{{ .Values.node.goquorum.resources.cpuLimit }}"
+              memory: "{{ .Values.node.goquorum.resources.memLimit }}"
+          volumeMounts:
+            - name: static-nodes
+              mountPath: /{{ .Release.Name }}/staticNode/
```

Comment:
Don't think the mountpaths should have variables. Just because it was done previously does not make it right. Please use static paths.